codec/raw/codec.go

```diff
@@ -0,0 +1,63 @@
+// Package raw implements IPLD's raw codec, which simply writes and reads a Node
+// which can be represented as bytes.
+//
+// The codec can be used with any node which supports AsBytes and AssignBytes.
+// In general, it only makes sense to use this codec on a plain "bytes" node
+// such as github.com/ipld/go-ipld-prime/node/basic.Prototype.Bytes.
+package raw
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+
+	ipld "github.com/ipld/go-ipld-prime"
+	cidlink "github.com/ipld/go-ipld-prime/linking/cid"
+)
+
+var (
+	_ cidlink.MulticodecDecoder = Decoder
+	_ cidlink.MulticodecEncoder = Encoder
+)
+
+// TODO(mvdan): make go-ipld-prime use go-multicodec soon
+const rawMulticodec = 0x55
+
+func init() {
+	cidlink.RegisterMulticodecDecoder(rawMulticodec, Decoder)
+	cidlink.RegisterMulticodecEncoder(rawMulticodec, Encoder)
+}
+
+type byteAccesor interface {
+	Bytes() []byte
+}
+
+func Decoder(am ipld.NodeAssembler, r io.Reader) error {
```

Comment:
Would prefer to call this `Decode` (minus the 'r') now.  (I've changed most of the other functions to this grammar in #143.)

Reply:
Ah, right, thanks for reminding me.